common/config/config.go

```diff
@@ -61,7 +61,7 @@ type AppConfig struct {
 	DBName string `mapstructure:"pg_db_name"`
 
 	RPCTimeout         Duration                     `mapstructure:"rpc_timeout"`
-	RPCMaxReqSize      int                          `mapstructure:"rpc_req_limit"`
```

Comment:
There are still a number of occurrences of "rpc_req_limit", like in `/home/jon/kwil/git/kwil-db/cmd/kwil-admin/nodecfg/toml.go` and some actual toml files.

Other thing is that `rpc_req_limit` was in v0.8, so any config files with this will have the setting ignored in 0.9.  That doesn't bother me too much, but ideally we'd keep recognizing it and have similar deprecation handling where we first process what we read from viper or whatever.

Reply:
Ah that's a good point. Ok, I will do that.